src/app/view/components/dot-device-selector/dot-device-selector.component.ts

```diff
@@ -21,8 +21,14 @@ export class DotDeviceSelectorComponent implements OnInit {
     ngOnInit() {
         this.options = this.dotMessageService
             .getMessages(['editpage.viewas.default.device'])
-            .mergeMap((messages: string[]) =>
-                this.dotDevicesService.get().pipe(map((devices: DotDevice[]) => this.setOptions(messages, devices)))
+            .mergeMap(() =>
+                this.dotDevicesService
+                    .get()
+                    .pipe(
+                        map((devices: DotDevice[]) =>
+                            this.setOptions(this.dotMessageService.get('editpage.viewas.default.device'), devices)
```

Comment:
Sooooooooo... we shouldn't been doing this

```this.dotMessageService.get('editpage.viewas.default.device')```

here, because in theory the ```.getMessages(['editpage.viewas.default.device'])``` should return that, but somehow is not working in FF and I don't have time right now to refactor the message service.